resources/css/app.css

```
@@ -1,30 +1,29 @@
-@layer tailwind-base, primevue, tailwind-utilities;
-
-@layer tailwind-base {
-    @tailwind base;
+@import "tailwindcss";
+@theme {
+    --font-sans:
+        Open Sans, ui-sans-serif, system-ui, sans-serif, "Apple Color Emoji",
+        "Segoe UI Emoji", Segoe UI Symbol, "Noto Color Emoji";
 }
+@custom-variant dark (&:where(.dark, .dark *));
+@import "tailwindcss-primeui";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)